### PR TITLE
Add Codeception task

### PR DESCRIPTION
**Pull request by adam-paterson**

### CODE REVIEW

**Comment by veewee** [comment]
Hi @adam-paterson,

Thanks for the codeception task!
Can you change the task to use the new `ProcessArgumentsCollection`:
https://github.com/phpro/grumphp/blob/master/src/GrumPHP/Collection/ProcessArgumentsCollection.php

**Comment by adam-paterson** [comment]
@veewee didn't notice that, amended. I'll squash if approved/

File: src/GrumPHP/Task/Codeception.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $files = $context->getFiles()->name('*.php');
+        if (0 === count($files)) {
+            return;
+        }
+
+        // We don't care about changed files here, we want to run the entire suit every time
```
Comment (aderuwe) [inline src/GrumPHP/Task/Codeception.php]: A bit of a nit-pick, but we've been removing these comments from the tasks, they're pretty much a copy/paste relic from the past... :)

File: grumphp.yml [hunk 2]
```diff
@@ -3,8 +3,4 @@ parameters:
   bin_dir: "./vendor/bin"
   git_dir: "."
   tasks:
-      phpcs:
-        standard: PSR2
-        ignore_patterns:
-          - "*Spec.php"
-      phpspec: ~
+      codeception: ~
```
Comment (aderuwe) [inline grumphp.yml]: Again, a nit-pick, but could you make sure this file ends in a new line?

Comment (aderuwe) [inline grumphp.yml]: Actually, I think this change is wrong and is causing the Travis build to fail - GrumPHP doesn't have any codeception tests, we do however want to be PSR2 compliant and run our specs. :)

**Comment by aderuwe** [comment]
We should probably also add codeception to the "suggest" section of composer.json.

**Comment by aderuwe** [comment]
:+1: feel free to squash the commits.

**Comment by adam-paterson** [comment]
@aderuwe Done!

**Comment by aderuwe** [comment]
Thanks!

**Pull request merged**